frontend/src/scenes/funnels/FunnelLineGraph.tsx

```diff
@@ -10,11 +10,11 @@ import { dayjs } from 'lib/dayjs'
 import { getFormattedDate } from 'scenes/insights/InsightTooltip/insightTooltipUtils'
 
 export function FunnelLineGraph({
-    dashboardItemId,
     inSharedMode,
     color = 'white',
+    showPersonsModal = true,
 }: Omit<ChartParams, 'filters'>): JSX.Element | null {
-    const { insightProps, insight } = useValues(insightLogic)
+    const { insightProps, insight, isViewedOnDashboard } = useValues(insightLogic)
```

Comment:
I think it may be better not to rely on this feature of the logic, and instead fully rely on `showPersonsModal`. That's because being viewed on a dashboard is just one of the cases where this should be hidden. The behavior we strictly need is for the persons modal to be disabled _when viewed in `InsightCard`_(and that may be on a dashboard or in saved insights), so a possibly clearer solution is to exclusively pass `showPersonsModal={false}` down when using the viz component in `InsightCard`.